### ISSUE TITLE
Lasso tool - Enable entire features selection

### ISSUE THREAD

**Issue by ignaciolep** [openstreetmap/iD#7375]
Since now iD can edit multiple selected features #7276, the shift-click-drag (lasso tool) should select the entire feature once all its nodes are selected, instead of only selecting nodes as now.

The use case is when the user needs to select multiple areas to change the tagging of them. For example consider selecting multiple areas tagged as "house" to change them to "plot" preset. Now the user has to shift-click over each one of the feature instead of using the lasso tool.

**Comment by ignaciolep** [comment]
Example of the use case:
![screenshot](https://user-images.githubusercontent.com/13091755/74932789-68302280-53c1-11ea-80be-533dacc8daa8.jpg)

**Comment by quincylvania** [comment]
I agree we should have a quicker way to select lines and areas, but it might be nice to also keep node-only selection as an option. Perhaps a second hotkey?

It's also a pain to do polygon-polygon intersection accurately and efficiently, but worth figuring out.

**Comment by ignaciolep** [comment]
> Perhaps a second hotkey?

Agree!

**Comment by 1ec5** [comment]
#8264 adds a keyboard shortcut to select the nodes’ parent ways, which you’d use after using the lasso tool.

**Comment by quincylvania** [comment]
> #8264 adds a keyboard shortcut to select the nodes’ parent ways, which you’d use after using the lasso tool.

This is a great new workflow, but I'd still like a way to lasso-select lines and areas directly.

**Comment by SomeoneElseOSM** [comment]
Interestingly, some seems to be asking for exactly this over at the help site: https://help.openstreetmap.org/questions/85519/how-do-i-lasso-select-buildings-only?page=1&focusedAnswerId=85898#85898 .
